src/generic/superlu_dist.c

```diff
@@ -1256,21 +1266,21 @@ void superlu_dist_global_matrix(int opt_flag, int allow_permutations,
   int_t *perm_r; /* row permutations from partial pivoting */
   int_t *perm_c; /* column permutation vector */
   int_t *etree;  /* elimination tree */
-  int_t job, rowequ, colequ, iinfo, need_value, i, j, irow;
+  int_t job, rowequ, colequ, iinfo, i, j, irow; /* , need_value */
   int_t m, n, nnz;
   int_t *colptr, *rowind;
-  int_t Equil, factored, notran, permc_spec, dist_mem_use;
+  int_t Equil, factored, notran, permc_spec; /*, dist_mem_use; */
   NCformat *Astore;
   NCPformat *ACstore;
   Glu_persist_t *Glu_persist;
-  Glu_freeable_t *Glu_freeable;
+  Glu_freeable_t *Glu_freeable=0;
 
   /* Other stuff needed by SuperLU */
-  double  *berr;
+  double  *berr=0;
```

Comment:
`NULL`?

Reply:
Done (also searched for other "=0" patterns and replaced a few more.

Done.